pymie/mie.py

```diff
@@ -785,22 +923,199 @@ def diff_abs_intensity_complex_medium(m, x, theta, ktd):
     I_perp = m* Et_phi * np.conj(Ht_theta)
     
     return I_par.real, I_perp.real
+
+def amplitude_scattering_matrix(m, x, thetas, coordinate_system = 'scattering plane', 
+                        phis = None):
+    """
+    Calculates the amplitude scattering matrix for an n-dim array of thetas 
+    (and phis if in cartesian coordinate system)
+    
+    Elements of the amplitude scattering matrix are arranged as:
+    [S2  S3]
+    [S4  S1]    
+    
+    Change of basis from scattering plane to lab frame cartesian is calculated 
+    by multiplying M*S*(M^-1) where S is the amplitude scattering matrix and 
+    M is the change of basis matrix. The change of basis matrix is:
+        
+    [cosphi  sinphi]
+    [sinphi -cosphi]
+    
+    This matrix is equal to it's inverse, so we can get the scattering matrix 
+    in the cartesian coordinate system by multiplying:
+    
+    [cosphi  sinphi] * [S2  S3] * [cosphi  sinphi]
+    [sinphi -cosphi]   [S4  S1]   [sinphi -cosphi]
+     
     
-def _amplitude_scattering_matrix(n_stop, prefactor, coeffs, theta):
+    see pages 35,51-53 in Annie Stephenson lab notebook #3 for details
```

Comment:
Let's file an issue to put the relevant info in the documentation, once we have documentation.  It will be much easier to look up there than to find your lab notebook.